test-test.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Please drop this script from PR.

If needed travis CI should be updated, there are specific requirements, we don't test everything.

* latest bandit once per py2/py3 (it's not needed for every distro)
* markdownlint is enough to run just once too
* pylint py2, freezed at F29 as newer pylint version stopped supporting py2
* pylint py3, tested on latest fedora only, no need to test each distro
* F32 should be able run only py3, we will replace F30 with F32 soon

Reply:
Removed.